client/web/src/components/widgets/inputs/DropdownInput.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+	<div class="dropdown-input">
+		<div class="dropdown-box" :style="{ minWidth: `${minWidth}px` }" @click="clickDropdownBox" data-hover-menu-spawner>
+			<Icon :class="'dropdown-icon'" :icon="activeEntry.icon" v-if="activeEntry.icon" />
+			<span>{{ activeEntry.label }}</span>
+			<Icon :class="'dropdown-arrow'" :icon="'DropdownArrow'" />
+		</div>
+		<MenuList
+			:menuEntries="menuEntries"
+			:activeEntry="activeEntry"
+			:defaultAction="setActiveEntry"
```

Comment:
Vue, by convention, prefers the use of `v-model` to have a two way binding. You trick Vue warning by setting `this.default` to `this.activeEntry`. Not really sure this works really.
Example with `v-model` was done with [SwatchPairInput.ts](https://github.com/GraphiteEditor/Graphite/blob/master/client/web/src/components/widgets/inputs/SwatchPairInput.vue#L12)